crates/router/src/connector/adyen/transformers.rs

```diff
@@ -259,30 +261,33 @@ pub struct RedirectionErrorResponse {
 
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct RedirectionResponse {
+pub struct NextActionResponse {
```

Comment:
Let's have separate Response for QR code and keep only the necessary fields which we receive in the QR response